README.md

```diff
@@ -224,6 +224,51 @@ See [using Ansible collections](https://docs.ansible.com/ansible/devel/user_guid
    > [!NOTE]
    Add this certificate to all RHTAS client nodes that use the `cosign` and `gitsign` binaries for signing and verifying artifacts.
 
+## Configuring a User-Provisioned MariaDB and Redis Instance for the RHTAS Ansible Collection
+    
+Prerequisites:
+    
+* An AWS account with necessary permissions to create RDS MariaDB and Elasticache Redis instances.
```

Comment:
On first use of an acronym, spell it out.

`An AWS account ...` --> `An Amazon Web Services (AWS)` account ...`

`... create RDS MariaDB ...` --> `... create releational database service (RDS) MariaDB ...`

Or, consider slightly rewording this sentence entirely.

`An Amazon Web Services (AWS) account with the ability to create MariaDB and Elasticache Redis instances.`